examples/chip-tool/commands/interactive/InteractiveCommands.cpp

```diff
@@ -308,7 +336,16 @@ bool InteractiveServerCommand::OnWebSocketMessageReceived(char * msg)
     }
 
     gInteractiveServerResult.Setup(isAsyncReport, timeout);
-    VerifyOrReturnValue(!isAsyncReport, true);
+
+    if (isAsyncReport)
+    {
+        if (gInteractiveServerResult.HasResults())
+        {
+            mWebSocketServer.Send(gInteractiveServerResult.AsJsonString().c_str());
+            gInteractiveServerResult.Reset();
```

Comment:
What happens if we have received part of the report (which is being chunked due to not fitting in a packet) before we get to this point, but the whole report?  I guess this only matters for non-single-attribute subscriptions and YAML tests should not do those?